controllers/discord.js

```diff
@@ -19,7 +19,7 @@ module.exports = class Discord extends require("./template.js") {
 			});
 		}
 
-		this.client = new (require("discord.js")).Client();
+		this.client = new (require("discord.js")).Client({disableMentions: "everyone"});
```

Comment:
in order to keep supibot syntax, add a space between curly braces and the identifiers

Reply:
fixed in 3139b7d